src/app/workloads/workloads.component.scss

```diff
@@ -25,4 +25,51 @@ p{
     p{
         font-size: 12px;
     }
-}
\ No newline at end of file
+}
+.mx-2:nth-of-type(2)
+{
+border-left: 1.4px solid rgb(175, 210, 236);
+padding-left: 15px;
+
+  }
+.mx-2:nth-of-type(1){
+    padding-right: 5px;
+}
+
+.my-auto:nth-of-type(2){
```

Comment:
what is my-auto?? don't overwrite bootstrap component

Reply:
My-auto is for vertical center !

Reply:
i am telling you to not to overwrite the bootstrap class
check out this(https://getbootstrap.com/docs/4.1/utilities/spacing/)